### PR TITLE
feat: JIRA-7386 Enforce Readonly properties when not written

### PR DESCRIPTION
**Pull request by owenvoke**

### CODE REVIEW

File: src/WorksomeRectorConfig.php [hunk 1]
```diff
@@ -50,6 +51,9 @@ public static function setup(RectorConfig $rectorConfig): void
         // PHP 8.0 set rules
         $rectorConfig->rule(ClassPropertyAssignToConstructorPromotionRector::class);
 
+        // PHP 8.1 set rules
```
Comment (lukeraymonddowning) [inline src/WorksomeRectorConfig.php:54]: What's the benefit of this line and line 51?

Comment (owenvoke) [reply src/WorksomeRectorConfig.php:54]: I guess nothing, I just copied the one above 🤷🏻 Lol

Comment (lukeraymonddowning) [reply src/WorksomeRectorConfig.php:54]: Why have any? If we're that intent on grouping, we could split by declarative methods? Could be wrong here, I just think they'll probably get ignored by somebody at some point anyway

Comment (owenvoke) [reply src/WorksomeRectorConfig.php]: Updated.

**Review by lukeraymonddowning — APPROVED** [review]

**Pull request merged**